packages/react-instantsearch/src/connectors/connectHighlight.js

```diff
@@ -36,7 +36,7 @@ export default createConnector({
 
   propTypes: {},
 
-  getProps() {
+  getProvidedProps() {
```

Comment:
I am moving this in another commit inside v2 if you don't mind since it's not directly linked to the current bug. So that we can release a new beta version and updated the website.